lib/iris/aux_factory.py

```diff
@@ -296,7 +296,6 @@ def _remap_with_bounds(self, dependency_dims, derived_dims):
                     # extra dimension to make the shape compatible, so
                     # we just add an extra 1.
                     shape.append(1)
-                nd_values = np.array(nd_values)
```

Comment:
I'm still not sure about this.  It makes me very nervous just whipping out a safety check (wow, how Sheldon is that??). 

Although since you have checked through the code path to make sure that there is no way for a non-array (i.e. a value) to appear here, I am a little happier about it.  

My trigger finger is itchy, but I also have ants in my pants about this a bit.  Not sure what to do...